tests/openwisp2/settings.py

```diff
@@ -82,8 +82,8 @@
 EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
```

Comment:
according to [openwisp-users docs](https://github.com/openwisp/openwisp-users#openwisp-users-auth-api) you need to enable this feature by adding `OPENWISP_USERS_AUTH_API=True` to this setting file

To enable Token creation, you will need to add `authtoken` to your `INSTALLED_APPS`. so add this after `rest_framework` in your `INSTALLED_APPS`  `'rest_framework.authtoken',`